acceptance/suites/tests/service_framework/reload_updated_crl.rb

```diff
@@ -0,0 +1,101 @@
+test_name '(SERVER-1380) Server loads updated CRL after being reloaded'
+
+puppetservice=options['puppetservice']
+cert_to_revoke="reload_updated_crl"
+server = master.puppet['certname']
+host_crl_file = master.puppet['hostcrl']
+ca_crl_file = master.puppet['cacrl']
+inventory_file = master.puppet['cert_inventory']
+
+teardown do
+  step 'Remove revoked CRL cert' do
+    on(master, puppet('cert', 'clean', cert_to_revoke),
+       {:acceptable_exit_codes => [0, 24]})
```

Comment:
@rlinehan I pushed up another commit, 8c30858, which just adds some comments about why 24 is tolerated as an exist code in a few places in the test.